x-pack/plugins/monitoring/public/alerts/types.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { CommonAlertStatus } from '../../common/types/alerts';
+
+export interface AlertsByName {
+  [name: string]: CommonAlertStatus;
+}
+
+export interface PanelItem {
+  id: number;
+  title: string;
+  width?: number;
+  content?: React.ReactElement;
+  items?: Array<ContextMenuItem | ContextMenuItemSeparator>;
+}
+
+export interface ContextMenuItem {
+  name: React.ReactElement;
+  panel?: number;
+  onClick?: () => void;
+}
+
+export interface ContextMenuItemSeparator {
+  isSeparator: true;
```

Comment:
Did you mean to do boolean here or the value?
```suggestion
  isSeparator?: boolean;
```

Reply:
I'm trying to match the type definition for EUI: https://github.com/elastic/eui/blob/master/src/components/context_menu/context_menu.tsx#L53